### ISSUE TITLE
Select - When value is an object, the aria-label should be human readable

### ISSUE THREAD

**Issue by halocline** [grommet/grommet#6393]
### Expected Behavior

When value is an object, the aria-label should be human readable, such as using the provided labelKey.

### Actual Behavior

Currently the label reads `aria-label="...; Selected: [object Object]`.

This probably involves work regarding the a11yTitle being set on StyledSelectDropButton.

### URL, screen shot, or Codepen exhibiting the issue

Jest test for `Select select an option with complex options 1` shows the aria-label being read as stated above.

[Snapshot test](https://github.com/grommet/grommet/blob/48663b7c6c520e00fa17d916b0a9d43d55c822b2/src/js/components/Select/__tests__/__snapshots__/Select-test.js.snap#L14367)

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version: v2.26.0
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by illiteratewriter** [comment]
Can I work on this?

**Comment by britt6612** [comment]
@illiteratewriter of course, feel free to let us know if you need any assistance.

**Comment by illiteratewriter** [comment]
Some snapshots are failing without me changing anything at all. mostly in the DateInput component.  What should I do about those? Is it because I am in a different time zone or something?

**Comment by jcfilben** [comment]
Hey @illiteratewriter someone else also ran into this issue, I am thinking it is timezone related and there are adjustments needed to our tests or directly to the DateInput component. **For now you can make commits with the `--no-verify` flag**. We have the environment set up so that the tests will also run on Github so we can confirm there that the tests are behaving as expected (other than having timezone related issues).

For debugging purposes what timezone are you working in?
